### ISSUE TITLE
Incompatible with upstream Grafana SDK changes?

### ISSUE THREAD

**Issue by slimsag** [K-Phoen/grabana#56]
Running go get on this as a dependency fails:

```
$ mkdir foo && cd foo/
$ go mod init foo.com/foo
go: creating new go.mod: module foo.com/foo
$ go get -u github.com/K-Phoen/grabana
go: github.com/K-Phoen/grabana upgrade => v0.6.3
go: github.com/grafana-tools/sdk upgrade => v0.0.0-20200326200416-f0431e44c1c3
go: github.com/gosimple/slug upgrade => v1.9.0
# github.com/K-Phoen/grabana/alert
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/alert.go:128:25: cannot use conditionOperator literal (type conditionOperator) as type sdk.AlertOperator in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:57:22: cannot use query literal (type query) as type sdk.AlertQuery in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:58:24: cannot use reducer literal (type reducer) as type sdk.AlertReducer in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:66:22: cannot use query literal (type query) as type sdk.AlertQuery in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:67:24: cannot use reducer literal (type reducer) as type sdk.AlertReducer in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:75:22: cannot use query literal (type query) as type sdk.AlertQuery in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:76:24: cannot use reducer literal (type reducer) as type sdk.AlertReducer in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:84:22: cannot use query literal (type query) as type sdk.AlertQuery in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:85:24: cannot use reducer literal (type reducer) as type sdk.AlertReducer in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:93:22: cannot use query literal (type query) as type sdk.AlertQuery in assignment
../pkg/mod/github.com/!k-!phoen/grabana@v0.6.3/alert/condition.go:93:22: too many errors
```

**Comment by slimsag** [comment]
Note: copying the grafana-sdk version from the go.mod in this repository works around the issue, so this is just a simple incompatibility issue with some upstream changes.

**Comment by K-Phoen** [comment]
> this is just a simple incompatibility issue with some upstream changes.

Correct :)

Should be fixed by #57, thanks for opening this issue!
